### PR TITLE
fix(Components): fix tests warnings

### PR DESCRIPTION
**Pull request by CernaianuMihai**
This PR is about fixing the unit tests warnings

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-236058

### CODE REVIEW

File: src/components/hoc/withAdapter.jsx [hunk 1]
```diff
@@ -87,7 +86,7 @@ export default function withAdapter(WrappedComponent, adapterFactory) {
        * @returns {Promise} A promise that resolves when the adapter finishes disconnecting
        */
       async function disconnect() {
-        setAdapterConnected(false);
+        global.testRendererAct(() => setAdapterConnected(false));
```
Comment (cipak) [inline src/components/hoc/withAdapter.jsx:89]: is this needed? the act() needs to be called after an await, but here it seems that setAdapterConnected() is called synchronously.

File: src/components/hooks/useOrganization.js [hunk 2]
```diff
@@ -20,18 +20,27 @@ export default function useOrganization(ID) {
...
-    };
-    const subscription = organizationsAdapter.getOrg(ID).subscribe(onOrganization, onError);
-
-    return () => {
-      subscription.unsubscribe();
-    };
+    let cleanup;
+
+    if (!ID) {
+      cleanup = undefined;
```
Comment (cipak) [inline src/components/hooks/useOrganization.js:26]: + `setOrganization({});`

File: src/storyshots.test.js [hunk 3]
```diff
@@ -35,14 +35,34 @@ initStoryshots({
...
+        if (snapshotFilename) {
+          expect(tree.toJSON())
+            .toMatchSpecificSnapshot(snapshotFilename);
+        }
+        if (story.id === 'platform-webex-avatar--invalid') {
+          /* eslint-disable no-console */
+          expect(console.error)
+            .toHaveBeenCalledTimes(1);
+          expect(console.error)
+            .toHaveBeenCalledWith('Could not find person with ID "user-7"');
```
Comment (cipak) [inline src/storyshots.test.js:61]: is breaking the line needed?

File: src/components/hoc/withAdapter.jsx (hunk 1 above)
Comment (CernaianuMihai) [reply src/components/hoc/withAdapter.jsx:89]: You are right, I'll remove it.

File: src/storyshots.test.js (hunk 3 above)
Comment (CernaianuMihai) [reply src/storyshots.test.js:61]: I'll remove them

**Review by cipak — APPROVED** [review]

**Pull request merged**